CHANGELOG.next.toml

```diff
@@ -431,3 +431,9 @@ message = "`Endpoint::immutable` now takes `impl AsRef<str>` instead of `Uri`. F
 references = ["smithy-rs#1984", "smithy-rs#1496"]
 meta = { "breaking" = true, "tada" = false, "bug" = false, "target" = "client" }
 author = "jdisanti"
+
+[[smithy-rs]]
```

Comment:
This is going to be in https://github.com/awslabs/smithy-rs/pull/1886 changelog? And the handlers used for the old service builder will still be stuck at 1.

Should we remove this entry?

Reply:
Ah, I forgot about that. Yeah, let's roll it up in there. Sorry @82marbag ❤️